Code/Editor/EditorFramework/Assets/Implementation/AssetBrowserModel.cpp

```diff
@@ -770,6 +797,8 @@ void ezQtAssetBrowserModel::FileSystemFileEventHandler(const ezFileChangedEvent&
 
   if (bFire)
   {
+    // QTimer::singleShot(500, [this]()
```

Comment:
Remove dead code ?

Reply:
Woopsi! Thanks for noticing. Is fixed now. I'll update clang tidy in a later commit to detect Qt pointers as pointer types.